test.py

```diff
@@ -1,2 +1,5 @@
+import os
+
 print("hello ")
 print("hii suraj")
+print(os.getcwd())
```

Comment:
this is not a proper way to use OS library.
Please avoid using print statements.
Use loggers instead.